Userland/Libraries/LibC/stdlib.cpp

```diff
@@ -844,9 +844,15 @@ char* mktemp(char* pattern)
 
 // https://pubs.opengroup.org/onlinepubs/9699919799/functions/mkstemp.html
 int mkstemp(char* pattern)
+{
+    return mkstemps(pattern, 0); // mkstemp is mkstemps with suffix_length 0.
+}
+
+// https://man7.org/linux/man-pages/man3/mkstemp.3.html
+int mkstemps(char* pattern, int suffix_length)
 {
     int fd = -1;
-    auto error = generate_unique_filename(pattern, [&] {
+    auto error = generate_unique_filename(pattern, static_cast<size_t>(suffix_length), [&] {
```

Comment:
We should probably guard this with an explicit check for negative values. A potential invalid value will get caught by `generate_unique_filename`, but I'm not sure whether we will trip any of the sanitizers using this cast regardless.

Reply:
I implemented the check as `VERIFY(suffix_length >= 0);`. I suppose that will satisfy the sanitizers.